tools/testsys/src/main.rs

```diff
@@ -1,12 +1,29 @@
-use clap::{Args, Parser, Subcommand};
+use anyhow::{Context, Result};
+use clap::{Parser, Subcommand};
+use delete::Delete;
+use install::Install;
+use logs::Logs;
+use model::test_manager::TestManager;
+use restart_test::RestartTest;
+use run::Run;
+use status::Status;
 use std::path::PathBuf;
 
-/// A program for running and controlling Bottlerocket tests through in a TestSys Kubernetes
-/// cluster.
+mod aws_resources;
+mod delete;
+mod install;
+mod logs;
+mod restart_test;
+mod run;
+mod status;
+
+/// A program for running and controlling Bottlerocket tests in a Kubernetes cluster using
+/// https://github.com/bottlerocket-os/bottlerocket-test-system
```

Comment:
Nit: if this message will be printed by the CLI when `cargo make <blah>` is run, I think instead of the link to testsys, you should just mention the name of the project?

```suggestion
bottlerocket-test-system
```